pkg/doc/verifiable/common.go

```diff
@@ -121,3 +121,51 @@ func stringSlice(values []interface{}) ([]string, error) {
 
 	return strings, nil
 }
+
+// decodeType decodes raw type(s).
+//
+// type can be defined as a single string value or array of strings.
+func decodeType(t interface{}) ([]string, error) {
```

Comment:
`decodeType()` was used for `Credential`, now is used by `Presentation` as well.